cmsdal/src/main/java/com/oneops/cms/dj/service/CmsRfcProcessor.java

```diff
@@ -512,44 +512,6 @@ public long createRfc(CmsRfcCI rfcCi) {
 	}
 
 
-	public void createRfcRaw(Collection<CmsRfcCI> rfcCis) {
```

Comment:
I know this function wasn't used right now, but idea was that if you have an operation that inserts a lot of rfcs this call would be most efficient since it groups all the inserts to the same table together, that achieves 10x speed of single rfc insert called in a cycle. 
So I would still keep it for future refactorings.